models/demos/mamba/tt/mamba_one_step_ssm.py

```diff
@@ -244,45 +209,31 @@ def forward(self, x):
         # repeat using mask+matmul instead of ttnn.repeat to avoid fallback
         C1 = self.transformer.repeat(
             C0,
-            memory_config=ttnn.L1_MEMORY_CONFIG,
-            compute_kernel_config=self.compute_kernel_config_mask,
-            core_grid=ttnn.CoreGrid(y=7, x=8),
+            memory_config=ttl.tensor.MemoryConfig(ttl.tensor.TensorMemoryLayout.INTERLEAVED, ttl.tensor.BufferType.L1),
         )
         ttnn.deallocate(C0)
 
-        # shard c
-        C2 = ttnn.to_memory_config(C1, memory_config=self.configs["sharded_dn"])
-        ttnn.deallocate(C1)
-
-        C3 = ttnn.mul(hidden_state1, C2, memory_config=self.configs["sharded_dn"])
+        C2 = ttnn.mul(hidden_state1, C1, memory_config=ttnn.L1_MEMORY_CONFIG)
         ttnn.deallocate(hidden_state1)
-        ttnn.deallocate(C2)
+        ttnn.deallocate(C1)
 
         # Reduction matmul
-        C3 = ttnn.to_memory_config(C3, memory_config=ttnn.L1_MEMORY_CONFIG)
-        C4 = self.transformer.reduce(
-            C3,
-            memory_config=ttnn.L1_MEMORY_CONFIG,
-            compute_kernel_config=self.compute_kernel_config_mask,
-            core_grid=ttnn.CoreGrid(y=7, x=8),
-        )  # b,n
-        ttnn.deallocate(C3)
-
-        # shard x, C
-        x = ttnn.to_memory_config(x, memory_config=self.configs["sharded_d"])
-        C5 = ttnn.to_memory_config(C4, memory_config=self.configs["sharded_d"])
-        ttnn.deallocate(C4)
-
-        # shard D
-        D = ttnn.to_memory_config(self.D, memory_config=self.configs["sharded_d"])
+        C3 = ttl.operations.primary.transformers.ssm_1d_sum_reduce(
+            C2,
```

Comment:
We need to add ssm_1d_sum_reduce to ttnn.   Ideally we do not use any ttl.operations.

Reply:
Should that be done here or in a follow-on PR?

Reply:
Instead of ttl.operations.primary.transformers.ssm_1d_sum_reduce
we need to use 
ttnn.experimental.operations.primary.transformers.ssm_1d_sum_reduce

Does this not work?

Reply:
That works, I'll update the calls to use the ttnn alias